pom.xml

```diff
@@ -45,6 +45,12 @@
 			<version>2.9.2</version>
 		</dependency>
 
+		<dependency>
```

Comment:
We shouldn't worry about importing sl4j, because every starter, like spring-boot-starter-web, depends on spring-boot-starter-logging. 

- [ ] Delete dependency